code/modules/jobs/job_types/chemist.dm

```diff
@@ -23,6 +23,12 @@
 
 	family_heirlooms = list(/obj/item/book/manual/wiki/chemistry, /obj/item/ph_booklet)
 
+	mail_goodies = list(
+		/datum/reagent/flash_powder = 15,
```

Comment:
```suggestion
		/datum/reagent/wittel = 15,
```
consider adding some of the other lavaland chems to this list as well

Reply:
one of those shifting recipe papers from that paper asteroid space ruin might also work here

Reply:
I removed wittel from the list for a reason, as I don't want to trivialize the amount of effort getting these secret recipes should entail. I'd be fine with including one of the recipe papers however.